evals/index.eval.ts

```diff
@@ -85,10 +99,22 @@ if (args.length > 0) {
   }
 }
 
-const models: AvailableModel[] =
+const DEFAULT_EVAL_MODELS = process.env.EVAL_MODELS
```

Comment:
can we instead have like `DEFAULT_EVAL_MODELS` and `EXPERIMENTAL_EVAL_MODELS`? the filter can then be parameterized too and it's easier to read